### PR TITLE
add support for ALTS transport

### PR DESCRIPTION
**Pull request by idank**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -222,6 +229,11 @@ func main() {
 		retcode = StatusInvalidArguments
 		return
 	}
+	if flALTS && flSPIFFE {
```
Comment (ahmetb) [inline main.go:232]: Probably can move this to L114 or so

Comment (idank) [reply main.go]: Done.

**Comment by idank** [comment]
Not sure it registered since I don't see the thread, but fixed.

**Comment by ahmetb** [comment]
Btw do you actually need this or are do you adding in case someone uses in the future?

**Comment by idank** [comment]
Need it asap, waiting for you to merge so I can import it into //third_party. ;)

**Comment by idank** [comment]
Anything else missing here?

**Comment by ahmetb** [comment]
My bad. I was waiting for tests to pass, then I forgot.

**Comment by idank** [comment]
Can you please release a new version and add a tag?

**Pull request merged**